Simplified/MyBooks/NYPLMyBooksDownloadCenter+AudiobooksDownload.swift

```diff
@@ -9,8 +9,54 @@
 import Foundation
 import NYPLAudiobookToolkit
 
+extension NYPLMyBooksDownloadCenter {
+  /// Create an audiobook object and add to download queue
+  @objc(downloadAudiobookForBook:)
+  func downloadAudiobook(for book: NYPLBook?) {
+    guard let book = book,
+          let url = self.fileURL(forBookIndentifier: book.identifier) else {
+      return
+    }
+    
+    AudiobookManifestAdapter.transformManifestToDictionary(for: book,
+                                                              fileURL: url) { json, decryptor, error in
+      if error == .none {
+        guard let audiobook = AudiobookFactory.audiobook(json, decryptor: decryptor) else {
+          Log.info(#file, "Audiobook initiate failed for book id - \(book.identifier)")
```

Comment:
this looks like a `Log.error` more than `info`. And I'd log `book.loggableShortString` instead of just the `id`